1_fetch/src/download_sb_data.r

```diff
@@ -5,13 +5,12 @@
   library(sbtools)
   library(whisker)
   
-  fetch_data <- function() {
-    project_output_dir <- '1_fetch/out'
+  fetch_data <- function(item_id, filepath_out) {
```

Comment:
Thanks for making these changes to the `fetch_data()` arguments, Ted. Looks like you just missed adding in the `filename` argument here in the function definition